app/views/all/export/by_month/conversions/projects/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<% content_for :primary_navigation do %>
+  <%= render partial: "shared/navigation/all_projects_primary_navigation" %>
+<% end %>
+
+<% content_for :page_title do %>
+  <%= page_title(t("export.by_month.conversions.index.title")) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h3 class="govuk-heading-l"><%= t("export.by_month.conversions.index.title") %></h3>
+    <%= t("export.by_month.conversions.index.body_html") %>
+  </div>
+</div>
+
+<table class="govuk-table" name="projects_table" aria-label="Transfers by month exports">
```

Comment:
`aria-label` should be 'Conversions by month exports'?

Reply:
Cut & paste 😬